src/test/java/Problem2Test.java

```diff
@@ -8,7 +8,12 @@
     // Okay to change
     @Test
     public void testBubbleSort() {
-        assertTrue(false);
         // add your tests here
+        int [] list = {5, 2, 1, 8, 7, 10, 22, 2, 4, 16}; //given list
+        int [] list2 = {1, 2, 2, 4, 5, 7, 8, 10, 16, 22}; //answer
+        Problem2.bubbleSort(list);
+        for (int i = 0; i < list.length; i++) {
+            assertEquals(list[i], list2[i]);
```

Comment:
add more test cases. 

what if input array is empty?
what if input array has one element?
what if input array has two different element?
what if input array has the same elements?
what if the input array is already sorted?
what if the input array is reversely sorted?

add test with these input and verify your code still works correctly.